test/blackbox-tests/test-cases/foreign-objects-exe.t

```diff
@@ -0,0 +1,68 @@
+----------------------------------------------------------------------------------
+Test suite for (foreign_objects ...) field.
+
+----------------------------------------------------------------------------------
+* (foreign_objects ...) is unavailable before Dune 3.5.
+
+  $ echo "(lang dune 3.4)" > dune-project
+  $ mkdir -p bin
+
+  $ touch calc.opam
+
+  $ cat >bin/dune <<EOF
+  > (executable
+  >  (public_name calc)
+  >  (foreign_objects add mul))
+  > EOF
+
+  $ dune build
+  File "bin/dune", line 3, characters 1-26:
+  3 |  (foreign_objects add mul))
+       ^^^^^^^^^^^^^^^^^^^^^^^^^
+  Error: 'foreign_objects' is only available since version 3.5 of the dune
+  language. Please update your dune-project file to have (lang dune 3.5).
+  [1]
```

Comment:
```suggestion
  Error: 'foreign_objects' is only available since version 3.5 of the Dune
  language. Please update your ``dune-project`` file to have (lang dune 3.5).
  [1]
```

Reply:
Test output doesn't get formatted with RST so there's no need to add such quotes.